pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -316,11 +323,14 @@ one = 'name is required. Run "rhoas kafka create --name my-kafka"'
 [kafka.create.error.conflictError]
 one = 'Kafka instance "{{.Name}}" already exists'
 
-[kafka.create.error.oneinstance]
-one = 'maximum number of allowed instances reached. Please review all instances that your user has access to and delete one or more instances before creating a new one.'
+[kafka.create.error.userInstanceType.notFound]
+one = 'Cannot fetch user allowed instance type"'
```

Comment:
I'm not sure this error message is correct. What is a "user allowed instance type"? I think this should be more like "Cannot fetch entitlements"

Reply:
Yes. I have struggled with finding good world/user facing message. Will change that